src/main/java/edu/ie3/datamodel/io/csv/BufferedCsvWriter.java

```diff
@@ -47,7 +47,7 @@ public BufferedCsvWriter(
                 baseFolder + File.separator + fileDefinition.getFilePath(), append),
             StandardCharsets.UTF_8));
     this.fileDefinition = fileDefinition;
-    this.quoted = quoted;
+    this.quoted = true;
```

Comment:
Please remove `quoted`. The constructor parameter as well as the attribute itself, as we intend to identify per String, if a quoting is necessary or not.